types/ObservationType.json

```
@@ -0,0 +1,106 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "/typesObservationType",
+    "description": "Abstract class that supports any observation or activity on a geospatial feature or crop",
+    "type": "object",
+    "required": [
+        "observationDate"
+    ],
+    "allOf": [
+        {
+            "$ref": "../types/ResourceType.json"
+        },
+        {
+            "not": {
+                "allOf": [
+                    {
+                        "required": [
+                            "feature"
+                        ]
+                    },
+                    {
+                        "required": [
+                            "featureId"
+                        ]
+                    }
+                ]
+            }
+        },
+        {
+            "not": {
+                "allOf": [
+                    {
+                        "required": [
+                            "holding"
+                        ]
+                    },
+                    {
+                        "required": [
+                            "holdingId"
+                        ]
+                    }
+                ]
+            }
+        }
+    ],
+    "properties": {
+        "observationDate": {
+            "type": "string",
+            "format": "date-time",
+            "description": "UTC Date (required) and time (optional) the sample was taken"
+        },
+        "phenomenonStartTime": {
+            "type": "string",
+            "description": "ISO UTC DateTime"
```

Should this have format "date-time"?
```suggestion
            "type": "string",
            "format": "date-time",
            "description": "ISO UTC DateTime"